contributing/releasing.md

```diff
@@ -19,6 +19,17 @@ is expected to do the following, in order of priority:
 If something is stopping the release engineer from achieving any of the above goals, the culprit
 code should be removed immediately from the release.
 
+If you are not able to cut a release by Wednesday afternoon this is *cause for concern*. Remember:
+landing the release is more important than landing any individual change. Revert suspect changes
+aggressively if something might be holding back the release.
+
+If you are going to be unavailable on the next working day and the release has not yet landed by
+Wednesday afternoon, you must hand the release off to the next release engineer before the end
```

Comment:
Do we have steps on how to "hand off a release"? A checklist or something maybe?

Reply:
Filed https://github.com/material-components/material-components-ios/issues/4980 to track writing this guide.

Reply:
Made the wording here more clear by adding 3PM as the cut-off point. Roll-backs tend to happen between 1-4 hours after a CL lands so this would result in the team being forced to stay after-hours to handle the fall-out.

Reply:
@yarneo 
> If you are blocking it after 3pm then the likelihood of a release starting and going out on Wedensday is low.